Server/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
what is this script for?

Reply:
waits for db service to start before starting backend